src/main/java/Group15/View/EditWorkoutView.java

```diff
@@ -25,18 +25,19 @@ public class EditWorkoutView
 
     public static Scene createScene(Workout workout)
     {
-        VBox layout = new VBox();
-        layout.setSpacing(20);
-        layout.setAlignment(Pos.CENTER);
+        BorderPane layout = new BorderPane();
```

Comment:
Can you explain your use of BorderPane, instead of a VBox?

Reply:
BorderPane is used to divide the layout into different sections and therefore I can set the bottom section as the same for every view